fixtures/input/nonempty/secret_path.yaml

```diff
@@ -8,3 +8,4 @@ metadata:
 type: Opaque
 data:
   SECRET_VAR: <path:secret/testing#secret-var-value>
```

Comment:
Can you add a file to nonempty that is using the secret version annotations as well?